### ISSUE TITLE
The capacitor controls should have a "clear capacitor" button

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#541]
The capacitor controls should have a "clear capacitor" button.  I can add a scaffolding for it, but I'll also assign @arouinfar since we'll need to design an icon.

**Comment by samreid** [comment]
I noted there are some icon ideas in the design doc.

**Comment by arouinfar** [comment]
@samreid if you need a place-holder, I would suggest using the reset icon, which is what is in the latest version of the mockups.

![image](https://user-images.githubusercontent.com/8419308/67720045-a16aff80-f998-11e9-9aef-c7bcbd9b0fdc.png)

Looking at the [4/9/19 meeting notes](https://docs.google.com/document/d/1bueLDNUhlGuYUMFVvQW2wlZMRknLG7ClZfVMd0GL6ag/edit#heading=h.ofumvjk8grba), another suggestion was to use a magnet with the with the ban symbol on top. There are font-awesome icons for the magnet (u+f076) and ban icon (u+f05e). The ban icon is pretty heavy, so we may want to just draw our own.

![image](https://user-images.githubusercontent.com/8419308/67723567-10992180-f9a2-11e9-950f-988a5a28c5f0.png)

From left to right
1. Stock font-awesome icons. Ban is too heavy, imo
2. Thinner ban icon + stock magnet
3. Thinner ban icon + magnet rotated 90*
4. Thinner ban icon + magnet rotated -45*

Personally I think option (4) is the most readable. @samreid @ariel-phet what do you think?

**Comment by ariel-phet** [comment]
I like the one on the left here. I think you could use the same button for the inductor and the capacitor actually. I view this as sort of "removing the electricity" button in layman's terms, and I think that is fine for both the inductor and capacitor. I don't think we need to get too technical that we are clearing the magnetic field, so I think a clear icon would be the best for both. (and I think this icon of the "bolt" does a nice job)

**Comment by samreid** [comment]
I also prefer the icon on the left.  I added it for both the capacitor and inductor.  @arouinfar can you please review on phettest?

**Comment by arouinfar** [comment]
@samreid this is looking good on master. I'll tag for design meeting for the team's sign off.

**Comment by samreid** [comment]
The design team agreed this icon works well for inductor and capacitor, closing.
